indico/modules/events/management/util.py

```diff
@@ -182,3 +186,40 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def get_unregistered_users(person_links):
```

Comment:
This is only used in this file. I'd underscore it and remove the docstring. It's not really useful elsewhere...

Considering how simple (one line) it is you could even just get rid of that function altogether. It's used only twice anyway.